src/widget/container/insets.rs

```diff
@@ -0,0 +1,74 @@
+use embedded_graphics::prelude::*;
+
+#[derive(Clone, Copy, Eq, PartialEq)]
+pub struct Insets {
```

Comment:
Not sure on naming here. I took this from the Flutter API name, but it's weird that it can be used as an inset (padding) and "outset" (margin).

Reply:
Ya it's sort of a "per-side" set of values